dlc/src/test/scala/org/bitcoins/dlc/BinaryOutcomeDLCWithSelfIntegrationTest.scala

```diff
@@ -46,6 +46,16 @@ class BinaryOutcomeDLCWithSelfIntegrationTest extends BitcoindRpcTest {
     } yield txid
   }
 
+  def waitUntilBlock(blockHeight: Int): Future[Unit] = {
```

Comment:
This seems like a pretty useful util that should probably be moved to `BitcoindRpcTestUtil`. I'd add a scaladoc saying that this will generate blocks until we reach the given `blockHeight`. This is useful for relative locktime schemes.